app/config/config_test.yml

```
@@ -1,6 +1,10 @@
 parameters:
     AdapterSecurityAdminClass:                LegacyTests\PrestaShopBundle\Mock\AdapterSecurityAdminMock
     prestashop.security.voter.product.class:  LegacyTests\PrestaShopBundle\Mock\PageVoter
+    resources_dir:                            "%kernel.project_dir%/tests/Resources"
+    translations_dir:                         "%resources_dir%/translations"
+    translations_theme_dir:                   "%resources_dir%%/themes"
+    translations_modules_dir:                 "%resources_dir%%/modules"
```

```suggestion
    translations_dir:                         "%kernel.project_dir%/tests/Resources/translations"
    translations_theme_dir:                   "%kernel.project_dir%/tests/Resources/themes"
    translations_modules_dir:                 "%kernel.project_dir%/tests/Resources/modules"
```
I'm not sure it was working because of double %

---

:+1: